apps/gauzy/src/app/pages/goals/keyresult-details/keyresult-details.component.html

```
@@ -201,11 +201,10 @@ <h6 class="subtitle mt-3">
       <div class="d-flex float-left">
         <div class="button-container">
           <button
-            class="mr-3"
+            class="mr-3 action"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)